### ISSUE TITLE
Wifi (Broadcom BCM43142) not supported in Live boot.

### ISSUE THREAD

**Issue by Jeremy7701** [linuxmint/lmde-4-cinnamon-beta#24]
Broadcom BCM43142 not detected in live boot.
Likely to be problematic for a range of Broadcom cards?
I proceeded with install to disk and on mandatory reboot, problem fixed.
Possible cause:- the install provides the `broadcom-sta-dkms` package.

**Comment by clefebvre** [comment]
Yes, we found that during QA but we're unable to find out the cause or a solution for it.

During live-boot LMDE installs broadcom-sta-dkms so this should work in live mode as well.... and then during the install live-installer installs it, so it works post-install.

The way this works in LMDE 4 is exactly the way it did in LMDE 3 but for some reason a reboot is needed now that we're on top of Buster..

**Comment by Jeremy7701** [comment]
Can I suggest an entry in the LMDE4 Release notes regarding this issue as it prevents some users trialling the live-boot version before an install or upgrade.

Not personally an issue as the laptop in question isn't presently used.

**Comment by clefebvre** [comment]
Yes. If you can manage to make the adapter work in live session let us know... the driver is installed, I tried to rmmod and modprobe but I wasn't successful.

**Comment by Jeremy7701** [comment]
I tried `dkms status` on both the live-boot and the installed versions.
Identical results shown for broadcom-sta
I don't understand dkms - except that it appears to be compiling source against the kernel.
It appears to be attempting to build the wl driver - tried rmmod and modprobe for wl.

I tried an old Mint 19.2 (MATE) magazine DVD and this also failed to recognise the wifi card on a live-boot, although I didn't try to install it.
So this seems to be a **very hard problem**!
